From 83dcfb7f301b6131fbc69ea5b4b0b773005b69a3 Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Thu, 22 Aug 2024 13:47:56 +0200
Subject: [PATCH] net-naming-scheme: disable NAMING_FIRMWARE_NODE_SUN

It seems that virtio devices always have "0" in
the firmware_node/sun. And because of that, udev will
always name the device ens0, which leads to collisions.
So let's disable it for now.

rhel-only: policy
Resolves: RHEL-55728
---
 src/shared/netif-naming-scheme.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shared/netif-naming-scheme.h b/src/shared/netif-naming-scheme.h
index c511ee86f5..c4e18bb035 100644
--- a/src/shared/netif-naming-scheme.h
+++ b/src/shared/netif-naming-scheme.h
@@ -85,7 +85,7 @@ typedef enum NamingSchemeFlags {
         NAMING_RHEL_9_4 = NAMING_RHEL_9_3,
         NAMING_RHEL_9_5 = NAMING_RHEL_9_4 & ~NAMING_BRIDGE_MULTIFUNCTION_SLOT,
 
-        NAMING_RHEL_10_0_BETA = NAMING_V255 | NAMING_FIRMWARE_NODE_SUN,
+        NAMING_RHEL_10_0_BETA = NAMING_V255,
 
         EXTRA_NET_NAMING_SCHEMES
 
